src/components/application_manager/test/event_engine_test.cc

```diff
@@ -108,6 +110,7 @@ class EventEngineTest : public testing::Test {
     smart_object_with_invalid_type["params"]["correlation_id"] = correlation_id;
     smart_object_with_invalid_type["params"]["function_id"] =
         hmi_apis::FunctionID::eType::VR_IsReady;
+    ON_CALL(mock_event_dispatcher_, remove_observer(_)).WillByDefault(Return());
```

Comment:
@ValeriiMalkov either return value or remove on call

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff